src/test/java/pages/booking/BookingHomePage.java

```diff
@@ -0,0 +1,72 @@
+package pages.booking;
+
+import driver.BasicDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import utils.DateParser;
+
+public class BookingHomePage {
+
+    WebDriver driver;
+    DateParser date = new DateParser();
+
+    public static final String BOOKING_SITE = "https://booking.com";
+    public static final String SEARCH_PLACE_INPUT = "//*[@id='ss']";
+    public static final String CALENDAR_RANGE = "//span[@class='sb-date-field__icon sb-date-field__icon-btn bk-svg-wrapper calendar-restructure-sb']";
+    public static final String EXTRA_PARAM_INPUT = "//label[@id='xp__guests__toggle']";
+    public static final String PLUS_MINUS = "//button[@aria-label = '%s: %s количество']";
+    public static final String CLOSE_ALL_POPUPS = "//span[contains(.,'другие варианты')]";
+    public static final String SEARCH_BUTTON = "//button[@data-sb-id='main']";
+    public static final String CURRENCY_BUTTON = "//button[@data-modal-aria-label  = 'Выберите валюту']";
+    public static final String LANGUAGE_BUTTON = "//button[@data-modal-aria-label  = 'Выберите язык']";
+    public static final String TOOLTIP_TEXT = "//div[@class= 'bui-tooltip__content']";
+    public static final String DATE_CALENDAR_SELECT = "//td[@data-date='%s']";
+
+    public BookingHomePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void goToBookingMain() {
+        driver.navigate().to(BOOKING_SITE);
+    }
+    public void searchCity (String cityName){
```

Comment:
reuse inputCity method and submitSearch

Reply:
I have method searchCity which includes input City + search(submit) and 2 separate methodes: method input and method search. They are used for different cases.